### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sophieschmieg**
Added a definition and a few projects I could think of off the top of my head.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +44,11 @@ of schemes that still need to be upgraded to post-quantum cryptography.
   Reading: [blog](https://security.apple.com/blog/private-cloud-compute/).
   Fully PQ: 😔.
 
+* **Apple/Google Exposure Notifications.**
+  Bespoke protocol.
+  Reading: [overview](https://www.google.com/covid19/exposurenotifications/)
+  Fully PQ: 😊. (With the expections of some signatures which could easily be changed to ML-DSA)
```
Comment (str4d) [inline README.md]:
```suggestion
  Fully PQ: 😊 (with the exception of some signatures which could easily be changed to ML-DSA).
```

File: README.md [hunk 2]
```diff
@@ -39,6 +44,11 @@ of schemes that still need to be upgraded to post-quantum cryptography.
   Reading: [blog](https://security.apple.com/blog/private-cloud-compute/).
   Fully PQ: 😔.
 
+* **Apple/Google Exposure Notifications.**
+  Bespoke protocol.
+  Reading: [overview](https://www.google.com/covid19/exposurenotifications/)
```
Comment (str4d) [inline README.md]: For formatting purposes:
```suggestion
* **Apple/Google Exposure Notifications.**  
  Bespoke protocol.  
  Reading: [overview](https://www.google.com/covid19/exposurenotifications/).  
```

**Review by str4d — APPROVED** [review]
ACK with suggestions.

**Review by bwesterb — APPROVED** [review]

**Comment by bwesterb** [comment]
Thanks!

**Pull request merged**